custom_components/recycle_app/api.py

```
@@ -79,6 +112,19 @@
         return (result["items"][0]["id"], result["items"][0]["names"][language])
 
     def get_recycling_parks(self, zip_code_id: str, language: str):
```

Ruff

Missing return type annotation for public function `get_recycling_parks`

(ANN201)